boards/arm/adi_eval_adin1110ebz/Kconfig.defconfig

```diff
@@ -0,0 +1,44 @@
+# ADI EVAL-ADIN1110EBZ board configuration
+
+# Copyright (c) 2024 BayLibre
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_ADI_EVAL_ADIN1110EBZ
+
+config BOARD
+	default "adi_eval_adin1110ebz"
+
+config SPI_STM32_INTERRUPT
+	default y
+	depends on SPI
+
+config MDIO_INIT_PRIORITY
+	default 81
+
+config PHY_INIT_PRIORITY
+	default 82
```

Comment:
Need to add `depends on` for their respective driver classes

Reply:
thanks, fixed